### ISSUE TITLE
Moving notes on nextcloud storage made them gone from all apps

### ISSUE THREAD

**Issue by pawel-t** [laurent22/joplin#722]
### Operating system
- Linux
- Android

### Application
- Desktop
- Mobile

I've tried to move my Joplin notes that are synchronized to nextcloud via WebDAV from main directory to subdirectory. Then I've changed path on linux/mobile apps. Once I've done synchronization - they are all gone and are not visible to applications. I've tried to import them manually (all md files), but as they are encrypted, I guess it's not possible. Is there any way to do it properly and restore my notes?

**Comment by pawel-t** [comment]
Actually something even more terrible happened. After few tries suddenly mobile/desktop app stopped to sync data at all. I mean on bother ever time I clicked synchronized I was getting "1 remote file created" while I didn't do any changed between those synchronizations. I've also noted that folder .sync and .resource were created but notes were never created on server side. 

In nginx logs I've spotted a lots of   
```
x.x.x.x - user [15/Aug/2018:19:45:30 +0100] "MKCOL /remote.php/webdav/notes/.sync/ HTTP/1.1" 405 258 "-" "okhttp/3.6.0"                                                                      
x.x.x.x- user [15/Aug/2018:19:45:30 +0100] "MKCOL /remote.php/webdav/notes/.resource/ HTTP/1.1" 405 258 "-" "okhttp/3.6.0"                                                                  
x.x.x.x - user [15/Aug/2018:19:45:30 +0100] "PROPFIND /remote.php/webdav/notes/d1eef8d349e14620900fe533cf31d195.md HTTP/1.1" 404 231 "-" "okhttp/3.6.0"  
```

So it looked like there was lack of synchronization between app and nextcloud. I've forced reindexing on nextcloud or even tried to change webdav folder location that was used to synchronize all data but it didn't help. I've cleared mobile app data and ... it started working. I'm reporting this issue as I want to find out what happened and how to prevent it in the future. Would be cool to also get to know how to copy/backup those notes - now I had only few but if I want to use it as my pocket notepad, I don't want to lose notes after like year once server goes down and I won't be able to restore it from saved files. Any idea how to achive this?
